packages/taquito-local-forging/src/taquito-local-forging.ts

```diff
@@ -3,13 +3,14 @@
  * @module @taquito/local-forging
  */
 
+import { InvalidBlockHashError } from './error';
 import { ForgeParams, Forger } from './interface';
 import { CODEC } from './constants';
 import { decoders } from './decoder';
 import { encoders } from './encoder';
 import { Uint8ArrayConsumer } from './uint8array-consumer';
 import { validateBlock, ValidationResult, InvalidOperationKindError } from '@taquito/utils';
-import { InvalidBlockHashError, InvalidOperationSchemaError } from './error';
+import { InvalidOperationSchemaError } from './error';
```

Comment:
this and `InvalidBlockHashError` in line 6 seems like they're from the same file. Are there any reason they are on different import statements?

Reply:
thanks for the catch put imported from taquito/core then switched it to error